src/sms/dtos/sms.params.dto.ts

```diff
@@ -16,4 +16,9 @@ export class SmsParamsDto {
     })
     @IsOptional() @IsNumber() readonly otp: number;
 
+    @ApiPropertyOptional({
```

Comment:
So, I think we don't want the authorization header as an api property on the DTO as this is supposed to just apply to the request body. If you want the requirement of an authorization header to appear in the swagger docs I think you can use ApiBearerAuth at the controller level: https://docs.nestjs.com/openapi/security

Reply:
Thanks, I'll take another look at this